examples/unet_segmentation_3d.py

```diff
@@ -134,30 +135,47 @@ def log_training_loss(engine):
                                      255, engine.state.epoch)
     engine.logger.info("Epoch[%s] Loss: %s", engine.state.epoch, engine.state.output[1])
 
-
-loader = DataLoader(ds, batch_size=20, num_workers=8, pin_memory=torch.cuda.is_available())
-val_loader = DataLoader(ds, batch_size=20, num_workers=8, pin_memory=torch.cuda.is_available())
 writer = SummaryWriter()
```

Comment:
This should be defined before the event function which uses it. It still works with Python's scoping behaviour but it makes more sense when defined first.

Reply:
Fixed